jinja2/qfdmo/adresse/detail.html

```diff
@@ -370,3 +370,9 @@ <h3 class="fr-text--sm fr-m-0 qfdmo-text-grey-425">Type d'enseigne</h3>
     >
     Proposer une modification
 </a>
+
+{% if request.user.is_staff %}
+    <div class="fr-m-0 fr-text-italic fr-text--xs qfdmo-text-center">
+        Pour le staff : <a target="_blank"  rel="noreferrer noopener" data-turbo=false href="{{ url('admin:qfdmo_displayedacteur_change', args=[adresse.identifiant_unique]) }}">Lien vers l'admin</a>
```

Comment:
```suggestion
        Pour le staff : <a target="_blank"  rel="noreferrer" data-turbo=false href="{{ url('admin:qfdmo_displayedacteur_change', args=[adresse.identifiant_unique]) }}">Lien vers l'admin</a>
```

Je pinaille, mais `target="_blank"` induit le `rel noopener` désormais, je pense qu'on ne supporte pas les versions de navigateur pour lesquelles ce n'est pas le cas. 

On peut donc l'omettre sans trop de soucis.

Reply:
Cf https://developer.mozilla.org/fr/docs/Web/HTML/Attributes/rel/noopener 

> Note : L'utilisation de target="_blank" sur des éléments <a> fournit désormais implicitement le même comportement pour l'attribut rel que si l'on avait utilisé rel="noopener", c'est à dire l'absence de mise en place de window.opener. Voir la [compatibilité navigateur](https://developer.mozilla.org/fr/docs/Web/HTML/Element/a#browser_compatibility) pour plus d'informations sur l'implémentation actuelle dans les navigateurs.

Reply:
Dans nos tests de compatibilité des navigateurs, on est un tout petit peu en dessous des nativateurs qui définissent noopener par défaut :

https://developer.mozilla.org/fr/docs/Web/HTML/Element/a#compatibilit%C3%A9_des_navigateurs

vs 

[static/to_compile/src/browser_check.ts](https://github.com/incubateur-ademe/quefairedemesobjets/blob/46afb431f5820a1f19d61933b6f8a832cfd118d7/static/to_compile/src/browser_check.ts#L3)

Est-ce qu'on relève un peu les versions des navigateur concernés et on supprime `noopener` quand il y a target="_blank" sur un lien ?

Reply:
pris en compte dans : https://github.com/incubateur-ademe/quefairedemesobjets/pull/811